org.eclipse.jdt.core.tests.model/src/org/eclipse/jdt/core/tests/dom/ASTConverterBugsTest.java

```diff
@@ -64,81 +61,6 @@ public static Test suite() {
 	return buildModelTestSuite(ASTConverterBugsTest.class);
 }
 
-protected void checkParameterAnnotations(String message, String expected, IMethodBinding methodBinding) {
```

Comment:
As we'd rather avoid diverging with upstream JDT at that point, I think that -as bad is it is in general- it's better to just duplicate it to the new test class.